mssql-plugin/src/main/java/io/cdap/plugin/mssql/SqlServerSinkDBRecord.java

```diff
@@ -55,7 +57,18 @@ protected void writeNonNullToDB(PreparedStatement stmt, Schema fieldSchema,
                                   String fieldName, int fieldIndex) throws SQLException {
     Object fieldValue = (fieldName != null) ? record.get(fieldName) : null;
```

Comment:
This can be moved to the switch GEOGRAPHY_TYPE and GEOMETRY_TYPE block below.

Reply:
done